blog/2024-05-07-weaviate-1-25-release/_core-1-25-include.mdx

```diff
@@ -4,8 +4,12 @@ Here are the release ⭐️*highlights*⭐️!
 
 ![Weaviate 1.25](./img/hero.png)
 
-- [**Dynamic Vector Index**](/blog/weaviate-1-25-release#dynamic-vectr-index) An index configuration that allows Weavaite to dyanamically switch from flat to HNSW as object count increases.
-- [**New Modules**](/blog/weaviate-1-25-release#new-modules) Loads of new modules that will allow you to use open source locally running and hosted embedding and language models!  
+- [**Dynamic Vector Index**](/blog/weaviate-1-25-release#dynamic-vector-index) Dynamically switch from flat indexes to HNSW to efficiently scale as your data grows.
+- [**Raft**](/blog/weaviate-1-25-release#raft) Improves schema management for faster updates and more reliable clusters. 
```

Comment:
I'm not sure the updates are faster as a whole (could confirm). The two benefits that I'm aware of are: allow concurrent schema operations, and higher fault tolerance in clusters. 

Background: Previously, any node being down would prevent schema operations, and only one schema operation could be performed at a time.